test/reporters/tests/reporters.spec.ts

```diff
@@ -44,15 +50,9 @@ test('JUnit reporter', async () => {
   const reporter = new JUnitReporter({})
   const context = getContext()
 
-  vi.mock('os', () => ({
-    hostname: () => 'hostname',
-  }))
-
-  vi.setSystemTime(1642587001759)
-
   // Act
   await reporter.onInit(context.vitest)
-  await reporter.onFinished(files)
+  await reporter.onFinished([])
```

Comment:
These reporter tests are rather difficult to update since it tries to mock out the entire Vitest instance.
What they tests are mostly only for `outputFile` option handling, so having something complex doesn't seem to worth it, so I simplified tests in `/reporters.spec.ts` in favor of integration testing in `/junit.test.ts`.